langchain4j-qianfan/src/main/java/dev/langchain4j/model/qianfan/QianfanStreamingLanguageModel.java

```diff
@@ -72,10 +84,10 @@ public void generate(String prompt, StreamingResponseHandler<String> handler) {
 
         CompletionRequest request = CompletionRequest.builder()
                 .prompt(prompt)
-                .top_k(topK)
-                .top_p(topP)
+                .topP(topK)
```

Comment:
should be `topK(topK)`.
BTW, shouldn't `topK` be `Integer`?

Reply:
Yes, topK is an integer,
The problem has been fixed